test/test_differentials.jl

```diff
@@ -32,9 +33,9 @@ using ManifoldDiff:
         @test differential_log_argument(M, p, p, X) == X
         differential_log_argument!(M, Y, p, p, X) == X
         @test Y == X
-        @test_broken differential_log_argument(M, p, q, X) == zero_vector(M, q)
-        differential_log_argument!(M, Y, p, q, X)
-        @test_broken Y == zero_vector(M, q)
+        @test differential_log_argument(M, p, q, X_qp) == -X
+        differential_log_argument!(M, Y, p, q, X_qp)
+        @test Y == -X
```

Comment:
@kellertuer could you check if this is correct? The result is now different.

Reply:
Oh, I will have to carefully check that.

Reply:
Yes that is correct. The intuition is as follows: `X_qp` is pointing from q to p, so if we move q into that direction, the result oof the log (X) is just shortened by the same amount, hence the result is -X (because we plug in. vector the same length as X).

So thanks for spotting this, it is correct now :)

Reply:
Thanks!